src/components/ToDoInfo/ToDoInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  title: string,
+  completed: boolean,
+  user: User | null,
+};
```

Comment:
don't repeat yourself
```suggestion
type Props = {
  todo: Todo;
};
```